packages/components/src/drop-zone/index.tsx

```diff
@@ -10,86 +10,14 @@ import { __ } from '@wordpress/i18n';
 import { useState } from '@wordpress/element';
 import { upload, Icon } from '@wordpress/icons';
 import { getFilesFromDataTransfer } from '@wordpress/dom';
-import {
-	__experimentalUseDropZone as useDropZone,
-	useReducedMotion,
-} from '@wordpress/compose';
+import { __experimentalUseDropZone as useDropZone } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
-import {
-	__unstableMotion as motion,
-	__unstableAnimatePresence as AnimatePresence,
-} from '../animation';
 import type { DropType, DropZoneProps } from './types';
 import type { WordPressComponentProps } from '../context';
 
-const backdrop = {
-	hidden: { opacity: 0 },
-	show: {
-		opacity: 1,
-		transition: {
-			type: 'tween',
-			duration: 0.2,
-			delay: 0,
-			delayChildren: 0.1,
-		},
-	},
-	exit: {
-		opacity: 0,
-		transition: {
-			duration: 0.2,
-			delayChildren: 0,
-		},
-	},
-};
-
-const foreground = {
-	hidden: { opacity: 0, scale: 0.9 },
-	show: {
-		opacity: 1,
-		scale: 1,
-		transition: {
-			duration: 0.1,
-		},
-	},
-	exit: { opacity: 0, scale: 0.9 },
-};
-
-function DropIndicator( { label }: { label?: string } ) {
-	const disableMotion = useReducedMotion();
-	const children = (
-		<motion.div
-			variants={ backdrop }
-			initial={ disableMotion ? 'show' : 'hidden' }
-			animate="show"
-			exit={ disableMotion ? 'show' : 'exit' }
-			className="components-drop-zone__content"
-			// Without this, when this div is shown,
-			// Safari calls a onDropZoneLeave causing a loop because of this bug
-			// https://bugs.webkit.org/show_bug.cgi?id=66547
-			style={ { pointerEvents: 'none' } }
```

Comment:
I wonder if this will need to be ported to CSS as well.

Reply:
Moved to the Sass file and kept the original comment